projects/Unstructured Supplemenrary  Service Data/ussdtim.py

```diff
@@ -0,0 +1,194 @@
+import random
+import time
+import sys
+
+print('Welcome To fastrack USSD Banking Project...')
+time.sleep(8)
+
+bank_list="""
+1. Access Bank
+2. Fidelity Bank
+3. Guarantee Trust Bank
+4. Heritage Bank
+5. Polaris Bank
+6. Stanbic IBTC
+7. Unity Bank
+8. Wema Bank
+"""
+
+gen_bvn = " "
+
+def BVN_checker( ):
+    global gen_bvn
+    bvn = [str(i) for i in range (5)]
+    gen_bvn= "".join(bvn)
+    print (gen_bvn)
+       				
+def open_acct( ):
+	global gen_bvn
+	print("Welcome to our online Account opening services.")
+	print("loading...")
+# creating an empty list to serve as a temporary place holder.
+	temp_storage= [ ]
+	f_name= input("Enter your first name:")
+	s_name= input ("Enter your second name:")
+	sex = input("Enter sex [M/F]:")
+	BVN_checker( )
+	temp_storage.append(f_name)
+	temp_storage.append(s_name)
+	temp_storage.append(sex)
+	temp_storage.append(gen_bvn)
+	details= " ".join(temp_storage)
+	print(details)
+	print("1. Press # to go back to options menu\n2. Press * to exit")
+	bck=input(":")
+	if bck=='#':
+	    options_menu( )
+	else:
+	    sys.exit( )
+	
+	exit( )
+	
+def upgrade_migrate( ):
+		print("Welcome to our online Upgrade/Migration services.\n 1. Ugrade\n 2. Migrate")
+		print("press # is go back to the Main Menu.")
+		prompt = int(input("Enter preferred Choice:"))
+		if prompt==1:
+			time.sleep(5)
+			print("Upgrading...")
+			exit( )
+		elif prompt == 2:
+			time.sleep(5)
+			print("Migrating...")
+			exit( )
+		elif prompt == "#":
+			options_menu( )
+		else:
+			sys.exit( )
+
+def balance ( ):
+	print("ACCOUNT\tBALANCE\n CHECKER")
+	print("press # is go back to the Main Menu.")
+	pin=input("Enter your 4 digit pin:")
+# isdigit( ) is used to  check for digits within a str while the nested if is used to make sure the user inputs 4 digits.
+
+###```i am to put the pin trial in a while loop```###REMINDER!!!
+
+	if len(pin)!=4:
+	   	print("Make sure its a 4digit pin.")
+	   	time.sleep(5)
+	   	balance( )
+	else:
+	   		    if pin.isdigit( )==True:

```

Comment:
Compare True Statements(Don't write extra statement)
```suggestion
	   		    if pin.isdigit( ):

```

Reply:
I don't  understand your suggestion here

Reply:
Bellow two statement perform same operation then why we need to put True
```python
if pin.isdigit(): 
```

```python
if pin.isdigit()==True: 
```

Reply:
Ok will change